app/models/cas/person.rb

```diff
@@ -1,14 +1,14 @@
 module Cas
-  class User < ApplicationRecord
-    ROLES = %w[admin editor writer].freeze
+  class Person < ApplicationRecord
+    ROLES = %w[admin editor writer visitor].freeze
```

Comment:
Pensando melhor, acho que o nome `visitor` aqui fica um pouco restrito... e se for um cliente? Se for um fornecedor? No caso da tua app, não seria visitor, seria mais um... customer?

Acho que o melhor nome seria `generic`. É tipo um coringa que serve pra tudo e aí no futuro tornamos mais específico. O que acha?

Reply:
☝️